propelauth_py/__init__.py

```diff
@@ -102,54 +102,54 @@
 )
 from propelauth_py.types.saml_types import SamlIdpMetadata
 
-from propelauth_py.validation import _validate_url
+from propelauth_py.validation import _validate_and_extract_auth_hostname
 
 class Auth:
     
-    def __init__(self, auth_url: str, integration_api_key: str, token_verification_metadata: Optional[TokenVerificationMetadata]):
-        self.auth_url = auth_url
+    def __init__(self, auth_hostname: str, integration_api_key: str, token_verification_metadata: Optional[TokenVerificationMetadata]):
+        self.auth_hostname = auth_hostname
```

Comment:
this is just a rename for clarity. unfortunately this makes review more annoying, but at least the naming is accurate